helm-chart/ray-cluster/templates/raycluster-cluster.yaml

```diff
@@ -299,12 +302,17 @@ spec:
         volumes: {{- toYaml .Values.worker.volumes | nindent 10 }}
         {{- end }}
         affinity: {{- toYaml .Values.worker.affinity | nindent 10 }}
-        {{ if .Values.worker.priorityClassName }}
+        {{- if .Values.worker.priorityClassName }}
         priorityClassName: {{- toYaml .Values.worker.priorityClassName | nindent 10 }}
         {{- end }}
-        {{ if .Values.worker.priority }}
+        {{- if .Values.worker.priority }}
         priority: {{- toYaml .Values.worker.priority | nindent 10 }}
         {{- end }}
+        {{- if .Values.worker.runtimeClassName }}
+        runtimeClassName: {{  .Values.worker.runtimeClassName | quote }}
+        {{- end }}
+        {{- if .Values.worker.topologySpreadConstraints }}
```

Comment:
Is this if statement needed. There's a check for topologySpreadConstraints online 199

Reply:
hmm not sure why it was added 😅 
I remove it